types/plotly.js-basic-dist/plotly.js-basic-dist-tests.ts

```diff
@@ -0,0 +1,20 @@
+import * as Plotly from 'plotly.js-basic-dist';
```

Comment:
Similarly here to test the typings for `plotly.js-basic-dist-min` and not those of `plotly.js-basic-dist`